.travis.yml

```diff
@@ -24,6 +19,7 @@ before_script:
 - if [[ "$TRAVIS_GO_VERSION" =~ 1.11 ]] && [ "$TRAVIS_TAG" != "" ]; then ./scripts/cross_compile.sh $TRAVIS_TAG; fi
 script:
 - go test -i $EXCLUDE_VENDOR
-- if [[ "$TRAVIS_GO_VERSION" =~ 1.11 ]]; then ./scripts/cov.sh TRAVIS; else go test -v -race $EXCLUDE_VENDOR; fi
+- go test -run=TestNoRace ./server
```

Comment:
Not following. Just place tests that we do not need to run under --race in their own file with !build constraint.

Reply:
I did not want to run the whole /server tests twice, it may take an additional 50s/1min, is that ok?